### ISSUE TITLE
Could not find implementation for ter-no-proto,  ter-no-script-url,  ter-no-self-compare

### ISSUE THREAD

**Issue by iarroyo5** [buzinas/tslint-eslint-rules#317]
### Bug Report
* **tslint version:** 5.9.0
* **tslint-eslint-rules version:** 5.1.0
* **Running TSLint via:** CLI

Could not find implementation for: 
- `ter-no-proto`
- `ter-no-script-url`
- `ter-no-self-compare`

**Actual configuration**
```javascript
module.exports = {
  extends: ['tslint-eslint-rules'],
  rules: {
    'ter-no-proto': true, // @FIXME [bug] Could not find implementation
    'ter-no-script-url': true, // @FIXME [bug] Could not find implementation
    'ter-no-self-compare': true, // @FIXME [bug] Could not find implementation
  }
}
```
^ I have more rules in the configuration and they work as expected. 

**Actual behaviour**
I get the following error:

<img width="679" alt="screen shot 2018-04-12 at 16 26 04" src="https://user-images.githubusercontent.com/5355188/38687288-3b77b722-3e6e-11e8-8b4e-0f0f5ff45a5e.png">

**Comment by mt-micky** [comment]
Still waiting: #307

**Comment by avrahamcool** [comment]
any news on this?

**Comment by jmlopez-rod** [comment]
Available in [5.2.0](https://github.com/buzinas/tslint-eslint-rules/blob/master/CHANGELOG.md)
